### DISCUSSION TITLE
Idea for documentation addition (styling sphinx-panels)

### DISCUSSION THREAD

**Discussion by donbowen** [executablebooks/meta#179] [Ideas]
Hi - I love the panels offered through sphinx-panels. They look great in jupyter books. 

One thing though, that took me an embarrassingly long time to figure out, and isn't obvious from [this page on card styling](https://sphinx-panels.readthedocs.io/en/latest/#card-styling) is that **directives for styling a card must begin on the line immediately after the `---` break between cards.**

## MWE to illustrate. This will work as expected:

```
````{panels}
:column: w-75 text-justified
Lorem ipsum and other stuff. Just keep typing. Shouldn't I copy the formal lorem ipsum? Meh. Nah. Too much work.
---
:column: w-25 text-center
See: This works!
\````
```
( Please ignore the slash in the last line - MD formatting issue)

## And this, with a single line break, fails:

```
````{panels}
:column: w-75 text-justified
Lorem ipsum and other stuff. Just keep typing. Shouldn't I copy the formal lorem ipsum? Meh. Nah. Too much work.
---

:column: w-25 text-center
See: This breaks!!
\````
```
( Please ignore the slash in the last line - MD formatting issue)

**Comment by choldgraf** [comment]
Ah yes, we should definitely make these kinds of things clear - would you like to make a PR suggesting a location to add this?

**Reply by donbowen** [reply]
Happy to do so! Submitted as PR #1110

**Reply by donbowen** [reply]
@choldgraf There was an issue with the CI/pre-commit check. Not sure what to do/if I should do anything on my end to resolve. (Sorry if this sounds silly, this is completely new to me, and my first PR ever.)
